### ISSUE TITLE
Fix epydoc pydoctor errors in remaining twisted parts

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8555]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8555|
|Type|enhancement|
|Created|2016-07-03 16:41:51Z|
|Branch|https://github.com/twisted/twisted/tree/8555-pydoctor-remaining|

protocols / internet / words / conch and other main modules have separate tickets for fixing epydoc

we now have various parts with a single error so this ticket is to do the final cleanup so that will no longer have pydoctor errors

<details><summary>Searchable metadata</summary>

```
trac-id__8555 8555
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__8555_pydoctor_remaining 8555-pydoctor-remaining
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467564111063031 1467564111063031
changetime__1467730123606446 1467730123606446
version__None None
owner__adiroiban adiroiban

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban removed owner|
|-|-|

This should be reviewed after all the other pydoctor tickets which are in the review queue.

1. L{pb.BrokerFactory} no longer exists and we have client and server factoris.  In one place I have replace it with an explicit reference to the two new implemetnation and the other places I have just call them generic `factory`.

2. For `twisted.internet.defer.TimeoutError:48 invalid ref to Deferred.setTimeout` we have 2 other tickets in the review queue ... so I will leave this error only after those tickets are reviewed.

----

Run the apidoc build

```
$ ./bin/admin/build-apidocs . apidocs/
```

besize zope errors, twisted.internet , twisted.words twisted.internet twisted.positioning (for which there are separate tickets) there should be no other errors.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @adiroiban|
|-|-|

Where is the branch for this?

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban removed owner|
|-|-|

:) On my local repo :)

I have pushed it now. Sorry.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_5"></a>|@rodrigc set owner to @adiroiban|
|-|-|

In twisted/runner/procmon.py, change C{True} to L{True} on the line you modified (might as well clean this up since you
are here).
Fix that, then go ahead and merge.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 86a4a9241a35dbc64ec0820d5e35bac02a123086
```
#!CommitTicketReference repository="" revision="86a4a9241a35dbc64ec0820d5e35bac02a123086"
Merge 8555-pydoctor-remaining: Fix pydoctor errors in remaining twisted code.

Author: adiroiban
Reviewer: rodrigc
Fixes: #8555
```
